.travis.yml

```diff
@@ -12,9 +12,9 @@ before_install:
       echo "Skipping CI; no changed formulae found in $TRAVIS_BUILD_DIR";
       travis_terminate 0;
     else
-      echo "Changed formulae are ${CHANGED_FORMULAE}";
-    fi
+      echo "Changed formulae are ${CHANGED_FORMULAE}"; fi
   - export HOMEBREW_REPOSITORY="$(brew --repo)"
+  - /usr/bin/yes | pip2 uninstall numpy
```

Comment:
Why would `numpy` installed into Python **2** affect this?

Reply:
Also, `pip uninstall -y numpy || true` would be the shortened command (ignoring version). See `pip uninstall -h`.

Reply:
Its installed by default and interferes with any `numpy` installation from brew. See the last build of `master`.

Reply:
> Also, pip uninstall -y numpy || true would be the shortened command (ignoring version)

You need to use `pip2` here as `pip` without a version is not known for some reason when using `xcode8.3`.

Reply:
> You need to use pip2 here as pip without a version is not known for some reason when using xcode8.3.

That's what "(ignoring version)" means.